test/integration/middleware.js

```diff
@@ -1,52 +1,57 @@
-"use strict";
+import assert from "assert";
+import mongoose from "mongoose";
+import request from "request";
+import sinon from "sinon";
+import { serve } from "../../src/express-restify-mongoose";
 
-const assert = require("assert");
-const mongoose = require("mongoose");
-const request = require("request");
-const sinon = require("sinon");
+import setupDb from "./setup";
 
 module.exports = function (createFn, setup, dismantle) {
-  const erm = require("../../src/express-restify-mongoose");
-  const db = require("./setup")();
+  const db = setupDb();
 
   const testPort = 30023;
   const testUrl = `http://localhost:${testPort}`;
   const invalidId = "invalid-id";
-  const randomId = mongoose.Types.ObjectId().toHexString();
+  const randomId = new mongoose.Types.ObjectId().toHexString();
   const updateMethods = ["PATCH", "POST", "PUT"];
 
-  describe("preMiddleware/Create/Read/Update/Delete - null", () => {
+  describe("preMiddleware/Create/Read/Update/Delete - undefined", () => {
     let app = createFn();
     let server;
     let customer;
 
-    beforeEach((done) => {
+    before((done) => {
       setup((err) => {
         if (err) {
           return done(err);
         }
 
-        erm.serve(app, db.models.Customer, {
-          preMiddleware: null,
-          preCreate: null,
-          preRead: null,
-          preUpdate: null,
-          preDelete: null,
```

Comment:
⚠️ Passing `null` will now crash, you must now pass either a middleware or middleware array.